src/protagonist/API/Features/DeliveryChannels/Requests/GetDeliveryChannelPolicies.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using API.Infrastructure.Requests;
+using DLCS.Model.Policies;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.DeliveryChannels.Requests;
+
+public class GetDeliveryChannelPolicies: IRequest<FetchEntityResult<IReadOnlyCollection<DeliveryChannelPolicy>>>
```

Comment:
Minor, but if I see `GetDeliveryChannelPolicies` I'd assume it's retrieving everything, whereas this is retrieving all policies for a channel

Reply:
`GetPoliciesForDeliveryChannel` ?